src/v/config/configuration.cc

```diff
@@ -2589,6 +2589,45 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::nullopt,
       {.min = 1.0, .max = 100.0})
+  , cloud_storage_inventory_based_scrub_enabled(
+      *this,
+      "cloud_storage_inventory_based_scrub_enabled",
+      "Scrubber uses the latest cloud storage inventory report, if available, "
+      "to check if the required "
+      "objects exist in the bucket.",
```

Comment:
```suggestion
      "objects exist in the bucket or container.",
```

Reply:
Are these suggestions to cover terms used in Azure?

Reply:
Sorry @abhijat I missed this question. Does this property only apply to Azure? If yes, then I think these should just use "container". If it also applies to AWS & GCP, then I think we should use "bucket or container".

Reply:
Thanks. Right now all of these properties only apply to AWS, but soon we will also support Azure and GCP, so I've made the changes to use `bucket or container` everywhere.